device/cuda/src/seeding/seed_finding.cu

```diff
@@ -127,6 +138,33 @@ __global__ void select_seeds(
 
 }  // namespace kernels
 
+vecmem::data::vector_buffer<device::prefix_sum_element_t> make_prefix_sum_buff(
+    const std::vector<device::prefix_sum_size_t> sizes,
+    const std::unique_ptr<vecmem::copy>& copy,
+    const traccc::memory_resource& mr) {
+
+    const device::make_buffer_result_t make_sum_result =
+        device::make_sum_buffer(sizes, copy, mr);
+    const vecmem::data::vector_view<const device::prefix_sum_size_t>
+        sizes_sum_view = make_sum_result.view;
+    const unsigned int totalSize = make_sum_result.totalSize;
+
+    // Create buffer and view objects for prefix sum vector
+    vecmem::data::vector_buffer<device::prefix_sum_element_t> prefix_sum_buff(
+        totalSize, mr.main);
+    copy->setup(prefix_sum_buff);
+    vecmem::data::vector_view<device::prefix_sum_element_t> prefix_sum_view =
+        prefix_sum_buff;
```

Comment:
In CUDA code you can skip creating this variable and just pass `prefix_sum_buff` directly to the kernel function. It gets cast into a view automatically. (Since the kernel function explicitly declares that it needs a view, and the buffer can be cast into a view.)